### ISSUE TITLE
Cannot find the contents which were starred

### ISSUE THREAD

**Issue by rhisss** [idno/Known#2037]
**While trying to do this:**
An user in a multi-user set up of Known, tries to find all the contents he has starred in the last month or last year or so far.

**I encountered this error:**
The user tries various ways but cannot find this.

**Some other notes**: We star a content so that we can come back to it later ( though bookmarks are there too BUT ?? _an user cannot apparently see a list of his private bookmarks_) or look back and read / see what we liked in the past / recent past and also enable others to look at my starred contents as a sort of recommendatiom.

**Comment by mapkyca** [comment]
<a href="https://github.com/idno/Known/issues/2037" class="u-in-reply-to"></a>You're not wrong, this is not currently possible. Search is based on objects, but "stars" are relationships not objects... so it doesn't appear in the search.

This is actually a fairly complex problem from a database query point of view.

It is however trivial to write a plugin to create such a page, and on large multiuser deploys, this is an often requested feature that is either built by them or paid for.

Duplicate of #941

**Comment by rhisss** [comment]
Thanks a lot. If it can be a "paid for" stuff what will be the approx cost and to approach whom? Voluntary or non-profit sites need many such basic plugins (there can be a debate what is "basic") which, unless being coming with core, may be difficult to maintain in future as the paid dev may fail to update the plugin with future versions of Known. Imho, getting a list of my starred content, having my own album, and ability to follow / befriend another member upon his consent ( or ability to block him) are needed very much for wide adoption of Known. Once wide adoption is done, there can be more payments for devs and overall betterment - but first new and free users need to be attracted (who are not essentially coders) away from Elgg or WP or the like. Many thanks.

**Comment by mapkyca** [comment]
<a href="https://github.com/idno/Known/issues/2037#issuecomment-369124480" class="u-in-reply-to"></a>PRs welcome! :)
